hoomd/hpmc/pytest/conftest.py

```diff
@@ -0,0 +1,1271 @@
+import pytest
+import hoomd
+import numpy as np
+
+
+@pytest.fixture(scope="function")
+def convex_polygon_integrator():
+    def make_shape():
+        return hoomd.hpmc.integrate.ConvexPolygon
+    return make_shape
+
+
+@pytest.fixture(scope="function")
+def convex_polygon_parameters():
+    def get_parameters():
+        return hoomd.hpmc._hpmc.PolygonVertices
+    return get_parameters
+
+
+@pytest.fixture(scope="function")
+def convex_polygon_valid_args():
+    def get_args():
+        args_list = [{'vertices': [(0, (0.75**0.5) / 2),
+                                   (-0.5, -(0.75**0.5) / 2),
+                                   (0.5, -(0.75**0.5) / 2)],
+                      'ignore_statistics': 0,
```

Comment:
Could we try one of these without `ignore_statistics` explicitly set. That should still work since it is optional. Same for `sweep_radius` I believe.